tests/all/integration/api/helpers/test_scheduled_jobs.py

```diff
@@ -29,3 +31,21 @@ def test_event_invoices_mark_due(self):
 
             self.assertEqual(status_new, "due")
             self.assertNotEqual(status_paid, "due")
+
+    def test_delete_ticket_holders_with_no_order_id(self):
+        """Method to test deleting ticket holders with no order id after expiry time"""
+        with app.test_request_context():
+            attendee_new_with_no_order_id = AttendeeFactory()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'attendee_new_with_no_order_id' is assigned to but never used](https://app.codacy.com/app/fossasia/open-event-orga-server/pullRequest?prid=4674573)